R/codemeta_description.R

```diff
@@ -48,134 +56,206 @@ additional_codemeta_terms <- function(){
   )
 }
 
-
+# codemeta_description ---------------------------------------------------------
```

Comment:
I haven't been good at that myself but the best would be to use `#'` to write comment, with `roxygen2` tags (but `@noRd`) rather than dashes.

Reply:
I am always using this kind of (non-roxygen) comments, just to separate the code sections visually. In RStudio, code sections separated like that can be folded and unfolded, which I like a lot as a feature:
 
![code_folding](https://user-images.githubusercontent.com/11964315/49502186-dc41e180-f874-11e8-93f1-40cae3f4f300.png)

Usually, I put roxygen comments below this separator line, such as in
https://github.com/KWB-R/kwb.utils/blob/master/R/dictionary.R

Reply:
aaah good point I had forgotten about the folding!